services/src/main/proto/com/github/rush/services/products_service.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package com.github.rush.services.products;
+
+option java_multiple_files = true;
+option java_package = "com.github.rush.services.products";
+
+// Service providing rpcs to find product.
+service ProductsService {
+  rpc FindProduct(ProductRequest) returns (ProductResponse);
+}
+
+message ProductRequest {
+  int32 id = 1;
+  string name = 2;
```

Comment:
let's leave only ID in request. At this moment we don't know which way we are going to request products. We can assume that we will need ID for sure. If we'll need something else then it's way a lot easier to add something to proto than taking care of removing something.